test/e2e/placement_pickall_test.go

```diff
@@ -1002,4 +1002,81 @@ var _ = Describe("placing resources using a CRP of PickAll placement type", func
 			ensureCRPAndRelatedResourcesDeleted(crpName, nil)
 		})
 	})
+
+	// TO-DO (chenyu1): modify the E2E setup to allow fixed node SKU assignment, so that the test suite can include
+	// a case where SKU-based scheduling can yield consistent non-empty results.
+	Context("with affinities, property selectors only (VM size), no matching clusters", Ordered, func() {
+		crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+		// Cross-ref with the setup script to ensure that this VM size is not used at all.
+		absentVMSize := "Standard_D8s_v3"
```

Comment:
Does this have to a real SKU name? Might be safer to make it something that is never going to appear in setup script

Reply:
Hi Wei! Yeah, for the E2E as there are Karpenter client libs in use for cost calculations we must use real SKU names. I have a PR in preparation that aims to move out property-related tests for simplicity reasons.